### DISCUSSION TITLE
Usage of Predicates with LINQ to AQL

### DISCUSSION THREAD

**Discussion by AdamJozwiak** [coronabytes/dotnet-arangodb#67] [Q&A] [answered]
Hi, I had problems where I wanted to use a **Predicate** in **Where** statement, but it could not parse it properly. Is predicate usage supported, or is it something that you could maybe expand your lib with?

**Comment by coronabytes** [comment, marked as answer]
predicate in the form of `Expression<Func<bool>>` should be supported, however it would best if you make a pull request with an unittest and said expression predicate.
